includes/class-amp-story-post-type.php

```diff
@@ -986,184 +986,21 @@ public static function get_fonts() {
-			],
-			[
-				'name'      => 'Montserrat',
-				'gfont'     => 'Montserrat:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'Noto Sans',
-				'gfont'     => 'Noto Sans:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'Open Sans',
-				'gfont'     => 'Open Sans:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'Open Sans Condensed',
-				'gfont'     => 'Open Sans Condensed:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'Oswald',
-				'gfont'     => 'Oswald:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'Palatino',
-				'fallbacks' => [ 'Palatino Linotype', 'Palatino LT STD', 'Book Antiqua', 'Georgia', 'serif' ],
-			],
-			[
-				'name'      => 'Papyrus',
-				'fallbacks' => [ 'fantasy' ],
-			],
-			[
-				'name'      => 'Playfair Display',
-				'gfont'     => 'Playfair Display:400,700',
-				'fallbacks' => [ 'serif' ],
-			],
-			[
-				'name'      => 'PT Sans',
-				'gfont'     => 'PT Sans:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'PT Sans Narrow',
-				'gfont'     => 'PT Sans Narrow:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'PT Serif',
-				'gfont'     => 'PT Serif:400,700',
-				'fallbacks' => [ 'serif' ],
-			],
-			[
-				'name'      => 'Raleway',
-				'gfont'     => 'Raleway:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'Roboto',
-				'gfont'     => 'Roboto:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'Roboto Condensed',
-				'gfont'     => 'Roboto Condensed:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'Roboto Slab',
-				'gfont'     => 'Roboto Slab:400,700',
-				'fallbacks' => [ 'serif' ],
-			],
-			[
-				'name'      => 'Slabo 27px',
-				'gfont'     => 'Slabo 27px:400,700',
-				'fallbacks' => [ 'serif' ],
-			],
-			[
-				'name'      => 'Source Sans Pro',
-				'gfont'     => 'Source Sans Pro:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'Tahoma',
-				'fallbacks' => [ 'Verdana', 'Segoe', 'sans-serif' ],
-			],
-			[
-				'name'      => 'Times New Roman',
-				'fallbacks' => [ 'Times New Roman', 'Times', 'Baskerville', 'Georgia', 'serif' ],
-			],
-			[
-				'name'      => 'Trebuchet MS',
-				'fallbacks' => [ 'Lucida Grande', 'Lucida Sans Unicode', 'Lucida Sans', 'Tahoma', 'sans-serif' ],
-			],
-			[
-				'name'      => 'Ubuntu',
-				'gfont'     => 'Ubuntu:400,700',
-				'fallbacks' => [ 'sans-serif' ],
-			],
-			[
-				'name'      => 'Verdana',
-				'fallbacks' => [ 'Geneva', 'sans-serif' ],
-			],
-		];
+		$file_content = file_get_contents( __DIR__ . '/data/fonts.json' );  // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
+		$google_fonts = json_decode( $file_content, true );
+		$fonts        = [];
```

Comment:
`$fonts` should still include default fonts from before like Arial, Verdana, etc.

Then, after adding the Google Fonts to the list, `$fonts` would need to be sorted by font name.

Reply:
This was not clear from the original issue.

Reply:
Sorry for the confusion then! Shall I update the issue to be more precise in general?